docs/usage/closed-prs.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Closed PRs
+description: How Renovate handles closed PRs
+---
+
+# Ignoring closed PRs
+
+By default renovate will ignore closed PRs and won't raise any PRs for same versions.
```

Comment:
I'm also missing a short explanation on _why_ you might need/want to label the PR to ignore it.

From issue https://github.com/renovatebot/renovate/issues/10674

> Ignoring PRs through another approach, instead of renames, since Azure DevOps does not allow you to rename a PR once the base branch has been deleted (which happens automatically in our case when abandoning PRs).